vendor/bat-native-ads/src/bat/ads/internal/database/tables/conversions_database_table.cc

```diff
@@ -259,6 +265,41 @@ void Conversions::MigrateToV10(mojom::DBTransaction* transaction) {
   util::CreateIndex(transaction, "creative_ad_conversions", "creative_set_id");
 }
 
+void Conversions::MigrateToV20(mojom::DBTransaction* transaction) {
+  DCHECK(transaction);
+
+  const std::string& temp_table_name = "creative_ad_conversions_temp";
+
+  const std::string& query =
+      "CREATE TABLE creative_ad_conversions_temp "
```

Comment:
nit: why not interpolate with `temp_table_name` from L271

Reply:
Migration will be refactored as part of https://github.com/brave/brave-browser/issues/14806